Sources/Core/JavaFileRenderer.swift

```diff
@@ -49,6 +49,56 @@ extension JavaFileRenderer {
         }
     }
 
+    func wrappedClassNameFromSchema(_ param: String, _ schema: Schema) -> String {
```

Comment:
Where is this function used? Please remove if it's not actually in use

Reply:
Sorry, I was messing around with an alternative approach to the type-adapter and this method is just a leftover from an abandoned approach. Removed.